CHANGELOG.md

```diff
@@ -1,3 +1,272 @@
+## 19.0.0 (December 5, 2024\)
+
+Below is a list of all new features, APIs, deprecations, and breaking changes. Read [React 19 release post](https://react.dev/blog/2024/04/25/react-19) and [React 19 upgrade guide](https://react.dev/blog/2024/04/25/react-19-upgrade-guide) for more information.
+
+> Note: To help make the upgrade to React 19 easier, we’ve published a react@18.3 release that is identical to 18.2 but adds warnings for deprecated APIs and other changes that are needed for React 19. We recommend upgrading to React 18.3.1 first to help identify any issues before upgrading to React 19.
+
+### New Features
+
+### React
```

Comment:
```suggestion
#### React
```

Here and below: I guess we need to use level 4 instead of level 3 for sub-headers of _New features_, _Breaking Changes_ so on. That will ensure the right semantic structure of the document.